.trunk/trunk.yaml

```diff
@@ -1,32 +1,32 @@
 version: 0.1
 cli:
-  version: 1.20.1
+  version: 1.22.0
```

Comment:
Please remove the changes to .trunk and protobufs/generated files. Protobufs need to go into its own repository.

Reply:
I reverted the changes made to the generated files. The pipeline fails because it is now missing those generated files. Just to be clear, you wanted me to revert changes made in the directory `src/mesh/generated/meshtastic`, right?

Reply:
yes, correct. you can ignore the failing pipeline, as long as your local copy compiles. Once the PR is polished, we'll merge the protobuf changes and the pipeline starts to work.